### PR TITLE
Assassination professions and mission

### PR DESCRIPTION
**Pull request by Termineitor244**
#### Summary
Content "Adds 2 new professions with a special mission"

#### Purpose of change
I wanted to add assassins to the game since some time ago, so... Here they are!

#### Describe the solution
Adds 2 new professions:

- Hitman/Hitwoman: An undercover agent adept at killing its targets at range with a pistol.
- Assassin: An undercover agent adept at killing its targets from close with knives.

Both are dressed in fancy clothes as part of their undercover activities, but have concealed weapons below these clothes, the cataclysm happened while they were on a new mission to kill a certain high-ranking individual... But go with caution, your target was not alone and you can no longer contact with your superiors.

#### Describe alternatives you've considered
To just add one profession maybe, or to have the mission destiny be a lab/office tower.

#### Testing
Started new games with the professions, and killed the target... Everything seems to be working alright!

#### Additional context
I think I have seen too much Spy x Family hahaha...

### CODE REVIEW

**Comment by sonphantrung** [comment]
Does the Bionic Assasin also count?

**Comment by Termineitor244** [comment]
> Does the Bionic Assassin also count?

It now does!

File: data/json/professions.json [hunk 1]
```diff
@@ -4564,5 +4577,111 @@
...
+          "fancy_sunglasses",
+          "platinum_watch",
+          "knit_scarf_loose",
+          "gloves_liner",
+          "polaroid_photo",
+          "suppressor",
+          "tele_sight_pistol"
+        ],
+        "entries": [
+          { "item": "glock_29", "ammo-item": "10mm_fmj", "charges": 15 },
```
Comment (TheShadowFerret) [inline data/json/professions.json:4606]:
```suggestion
          "polaroid_photo"
        ],
        "entries": [
          { "item": "glock_29", "ammo-item": "10mm_fmj", "charges": 15, "contents-item": [ "tele_sight_pistol", "suppressor" ] },
```
So it spawns attached, like Military Recruit's strap and holographic sight are

Comment (Termineitor244) [reply data/json/professions.json:4606]: It's not possible, the holster is small so the pistol does not fit with the mods attached (I tried hahaha). I though of increasing the holster pocket, but I'm not sure about that.

**Pull request merged**